### DISCUSSION TITLE
Select lines in a note, combine into paragraph?

### DISCUSSION THREAD

**Discussion by LorenAmelang** [TriliumNext/Trilium#1364] [Ideas]
I often paste a paragraph of text copied from a pdf, or a text file with line breaks, into a note, and get separate lines with the larger paragraph spacing. Frustrating when they are much shorter than the note width, and even worse when they are a bit too wide and each one wraps. Sometimes I'd like to keep the breaks and just narrow the spacing, which I could do by manually deleting each `<p>` and adding a `<br>` - if I had the patience. Sometimes I'd like to just delete all of the `<p>` breaks and make a continuous paragraph. 

Is there some way to select a group of lines and delete all the `<p>` breaks (ensuring a space remains between each one)? Or replace them with `<br>` breaks? Could there be?

**Comment by avinashkanaujiya** [comment]
Are you talking about removing formatting from the copied text?

**Reply by LorenAmelang** [reply]
I guess you could say that. But I'm not sure that it really conveys the important part. Like I have a hotkey that "removes formatting" from my clipboard content, but it only removes fonts and effects, it does not remove newlines or prevent `<p>` spacing in the pasted text. I usually don't know how the paste will look until it is already in Trilium. And I don't necessarily want to remove _all_ the paragraph breaks. The best way I can think of to do what I want would be to select a group of lines and have a hotkey or option to join them into a single paragraph. And another key or option to change all the contained `<p>` breaks to `<br>` breaks.

**Reply by avinashkanaujiya** [reply]
Yes, now i understand what you are saying.
You mean when you are pasting something into something (say a bullet list) you want pasted text to be a bullet and not appear separately as a paragraph.

**Reply by LorenAmelang** [reply]
I've never tried pasting into a list. My concern is this kind of thing:
![Trilium format](https://user-images.githubusercontent.com/1297090/97787868-ed372280-1b71-11eb-91a2-975052fcedeb.JPG)

I copy some text that includes line breaks, paste it into a note as a regular paragraph, and it gets a new paragraph for each line. Sometimes I want to keep the line breaks, but as Shift+Enter `<br>` breaks. Sometimes I want it to reflow without breaks. 

I usually can't tell what I will get until I have pasted it into Trilium, so the best solution would be a way to select some lines and "Collect into single paragraph", or "Reflow into single paragraph".
